### PR TITLE
INFRA-271: Added 'initializer' project type.

### PR DESCRIPTION
**Pull request by enyachoke**
…ion)

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  openmrscd:
+    image: mekomsolutions/openmrscd:latest
+    container_name: "openmrscd"
+    ports:
+      - "8080:8080"
+    volumes:
+     - ./node-scripts:/opt/node-scripts
+     - ./jenkins/jenkins_home:/var/jenkins_home
+     - ~/.m2:/var/jenkins_home/.m2
+     - ./data/app_data:/var/lib/openmrs_cd/app_data
```
Comment (mks-d) [inline docker-compose.yaml:13]: @enyachoke thanks for that!

File: README.md [hunk 2]
```diff
@@ -109,6 +109,10 @@ docker run --name openmrscd  -p 8080:8080 \
   mekomsolutions/openmrscd:latest
 ```
 
+### Using docker-compose
+
+```docker-compose up```
+
```
Comment (mks-d) [inline README.md:115]: @rbuisson I guess the landing README will have to be reviewed to put this in the right place?

**Review by mks-d — COMMENTED** [review]
Thanks @enyachoke, great work.

I'm mostly good with it, I guess we just need to review the landing README.

**Review by rbuisson — CHANGES_REQUESTED** [review]
Thanks!
Could you update the [config.xml](https://github.com/mekomsolutions/openmrs-cd/blob/fd8667ad248fe6387de8f3fb507b2d4cfc6e8636/jenkins/jenkins_home/jobs/pipeline1/config.xml#L25) to add the new 'initializer' type.

Otherwise, looks good (and works good) 👍🏻

**Review by rbuisson — APPROVED** [review]
Thanks!

**Pull request merged**